src/python_testing/matter_testing_support.py

```diff
@@ -878,6 +879,45 @@ def get_test_desc(self, test: str) -> str:
         except AttributeError:
             return test
 
+    # Sends an out-of-band command to a Matter app
+    def write_to_app_pipe(self, command):
```

Comment:
Please enforce that `command` is a dict, so that arbitrary commands do not occur.